notebook/agenteval_cq_math.ipynb

```diff
@@ -115,44 +187,16 @@
   {
    "cell_type": "code",
    "execution_count": 3,
-   "metadata": {
-    "id": "9XAeyjd11_BK"
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "critic = autogen.AssistantAgent(\n",
-    "    name=\"critic\",\n",
-    "    llm_config={\"config_list\": config_list},\n",
-    "    system_message=\"\"\"You are a helpful assistant. You suggest criteria for evaluating different tasks. They should be dinstinguishable, quantifieable and not redundant.\n",
-    "    Convert the evaluation criteria into a dictionary where the keys are the criteria.\n",
-    "    The value of each key is a dictionary as follows {\"description\": criteria description , \"accepted_values\": possible accepted inputs for this key}\n",
-    "    Make sure the keys are criteria for assessing the given task.  \"accepted_values\" include the acceptable inputs for each key that are fine-grained and preferably multi-graded levels. \"description\" includes the criterion description.\n",
-    "    Return the dictionary.\"\"\",\n",
-    ")\n",
-    "\n",
-    "critic_user = autogen.UserProxyAgent(\n",
-    "    name=\"critic_user\",\n",
-    "    max_consecutive_auto_reply=0,  # terminate without auto-reply\n",
-    "    human_input_mode=\"NEVER\",\n",
-    "    code_execution_config={\n",
-    "        \"use_docker\": False\n",
-    "    },  # Please set use_docker=True if docker is available to run the generated code. Using docker is safer than running the generated code directly.\n",
-    ")\n",
-    "\n",
-    "\n",
-    "def ask_critic(message):\n",
-    "    \"\"\"\n",
-    "    Initiate a chat with the critic user and return the last message received from the planner.\n",
-    "\n",
-    "    Args:\n",
-    "    - message (str): The message to be sent to the critic user.\n",
-    "\n",
-    "    Returns:\n",
-    "    - str: The content of the last message received.\n",
-    "    \"\"\"\n",
-    "    critic_user.initiate_chat(critic, message=message)\n",
-    "    # return the last received from the planner\n",
-    "    return critic_user.messagelast_message()[\"content\"]"
+    "def remove_ground_truth(test_case):\n",
```

Comment:
the markdown cell above this code seems to be stale. Make sure that the notebook markdown cells still make sense after the edits you did.